flappy_planes.html

```diff
@@ -1,31 +1,50 @@
+<!--
+    Contains all the UI elements for:
+    - Start screen with control selection
+    - Game canvas
+    - Score display
+    - Game over menu
+-->
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Flappy Planes</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="flappy_styles.css">
+    <link rel="stylesheet" href="styles.css">
```

Comment:
You should add a link to flappy_styles.css since that is the CSS file related to flappy planes. While style.css is for the top bar